dictionary.txt

```diff
@@ -866,7 +866,10 @@ _j	ⱼ
 \lrblacktriangle	◢	
 \llblacktriangle	◣	
 \ulblacktriangle	◤	
-\urblacktriangle	◥	
+\urblacktriangle	◥
```

Comment:
Looks good, there's a tab missing at the end of this line, though

```suggestion
\urblacktriangle	◥	
```

Reply:
Done,